source/expressions.tex

```diff
@@ -7579,17 +7857,26 @@
 $o$ is constexpr-referenceable from $P$.
 
 \pnum
+\indextext{contract evaluation semantics!ignore}
 A variable \tcode{v} is \defn{constant-initializable} if
 \begin{itemize}
 \item
 the full-expression of its initialization is a constant expression
-when interpreted as a \grammarterm{constant-expression},
+when interpreted as a \grammarterm{constant-expression}
+with all contract assertions
+using the ignore evaluation semantic\iref{basic.contract.eval}, and  %JMB: Should we remove this "and", there are 3 bullets?
```

Comment:
I hate the fact we have notes in the middle of the sentence.  Nothing we can do about that right now, though.

Yes, drop the "and".

Reply:
done